test/Microsoft.Azure.Functions.Worker.Extensions.OpenApi.Tests/OpenApiHttpTriggerContextTests.cs

```diff
@@ -151,7 +175,7 @@ public async Task Given_Type_When_GetOpenApiSpecVersion_Invoked_Then_It_Should_R
         public async Task Given_Type_When_GetOpenApiSpecVersion_Invoked_Then_It_Should_Return_Result(string format, OpenApiFormat expected)
         {
             var location = new FileInfo(Assembly.GetExecutingAssembly().Location).Directory.FullName;
-            var context = new OpenApiHttpTriggerContext();
+            var context = new OpenApiHttpTriggerContext(null);
```

Comment:
If we change the constructor parameter to be optional, we don't need to change this part.